protocol.go

```diff
@@ -155,3 +157,39 @@ func (r *errorTranslatingReceiver) Receive(msg any) error {
 func (r *errorTranslatingReceiver) Close() error {
 	return r.fromWire(r.Receiver.Close())
 }
+
+func sortedAcceptPostValue(handlers []protocolHandler) string {
+	var max int
```

Comment:
nit: should this be sum?

Reply:
It's the maximum possible size of the union, which is the sum of the individual set sizes.